certification/pyxis/submit_test.go

```diff
@@ -28,7 +28,14 @@ var _ = Describe("Pyxis Submit", func() {
 			It("should switch to In Progress", func() {
 				certResults, err := pyxisEngine.SubmitResults(ctx, &CertificationInput{
 					CertProject: &CertProject{CertificationStatus: "Started"},
-					CertImage:   &CertImage{},
+					CertImage: &CertImage{
```

Comment:
creating of this object could perhaps be factored into a func testCertImage() CertImage ?

Reply:
Maybe in a future PR. This works as is, and we need this fix in rather soon. It doesn't affect the functionality of the point of this PR.